{"name": "test_with_normal", "status": "broken", "statusDetails": {"message": "_pytest.fixtures.FixtureLookupError: ('self', <FixtureRequest for <Function test_with_normal>>)", "trace": "file E:\\python_project\\pythoncode\\test_allure\\test_allureseverties.py, line 11\n  @allure.severity(allure.severity_level.NORMAL)\n  def test_with_normal(self):\nE       fixture 'self' not found\n>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nE:\\python_project\\pythoncode\\test_allure\\test_allureseverties.py:11"}, "start": 1591981845480, "stop": 1591981845480, "uuid": "14756da5-acad-4e64-a808-45677d91f507", "historyId": "01e60880dab162942f77b1d4950ecb90", "testCaseId": "4f1de3e98a9229195d07ca2ca3b4685e", "fullName": "test_allureseverties#test_with_normal", "labels": [{"name": "severity", "value": "normal"}, {"name": "suite", "value": "test_allureseverties"}, {"name": "host", "value": "DESKTOP-UGIE581"}, {"name": "thread", "value": "12356-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_allureseverties"}]}